types/rlp-encoding/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for rlp-encoding 3.0
+// Project: https://github.com/fanatid/rlp-encoding
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+/**
+ * Encode value to [RLP](https://ethereum.org/en/developers/docs/data-structures-and-encoding/rlp).
+ *
+ * @param value The value to encode.
+ * @param buffer The buffer to encode the `value` to. If not provided, a new `Buffer` will be allocated.
```

Comment:
```suggestion
 * @param [buffer] The buffer to encode the `value` to. If not provided, a new `Buffer` will be allocated.
```

Reply:
This isn't necessary, typescript already makes sure to allow the according types. And this won't be rendered by quick hints by IDE. So I see no point in doing this.